File: 15-day/volumeCalculator.py
from math import pi

class cylinder():
    def __init__(self):
        self.height = float(input('Cylinder height? '))
        self.ratio = float(input('radius of the cylinder bases? '))

    def getVolume(self):
        return pi * self.ratio ** 2 * self.height

class cube ():
    def __init__(self):
        self.side = float(input('Side of the cube? '))
    def getVolume(self):
        return self.side ** 3

class sphere():
    def __init__(self):
        self.ratio = float(input('Ratio of the sphere? '))
    def getVolume(self):
        return (4/3) * pi * self.ratio ** 3

    option = input('Select: cylinder, cube, sphere ').lower()
    if option == 'cylinder':
        cylinder = cylinder()
        print('The volume of the cylinder is: {}'.format(round(cylinder.getVolume(), 2)))
    elif option == 'cube':
        cube = cube()
        print('The volume of the cube is: {}'.format(round(cube.getVolume(), 2)))
    elif option == 'sphere':
        sphere = sphere()
        print('The volume of the sphere is: {}'.format(round(sphere.getVolume(), 2)))
    else:
        print('Unrecognizable figure')

File: 15-day/volumeCalculator.js
class cylinder {
  constructor() {
    this.height = parseFloat(prompt('Cylinder height? '))
    this.ratio = parseFloat(prompt('Cylinder radius? '))
  }
  getVolume() {
    return Math.PI * Math.pow(this.ratio, 2) * this.height
  }
}

class cube {
  constructor() {
    this.side = parseFloat(prompt('Side of the cube? '))
  }
  getVolume() {
    return Math.pow(this.side, 3)
  }
}

class sphere {
  constructor() {
    this.ratio = parseFloat(prompt('Ratio of the sphere? '))
  }
  getVolume() {
    return (4 / 3) * Math.PI * Math.pow(this.ratio, 3)
  }
}

const option = prompt('Select: cylinder, cube, sphere ').toLowerCase()
switch (option) {
  case 'cylinder':
    const volumeCylinder = new cylinder()
    alert(`The volume of the cylinder is: ${volumeCylinder.getVolume().toFixed(2)}`)
    break
  case 'cube':
    const volumeCube = new cube()
    alert(`The volume of the cube is: ${volumeCube.getVolume().toFixed(2)}`)
    break
  case 'sphere':
    const volumeSphere = new sphere()
    alert(`The volume of the sphere is: ${volumeSphere.getVolume().toFixed(2)}`)
    break
}